apps/desktop/desktop/src/main/java/bisq/desktop/main/content/settings/bisq_connect/BisqConnectView.java

```diff
@@ -0,0 +1,248 @@
+import bisq.desktop.components.controls.Switch;
+import bisq.desktop.components.overlay.Popup;
+import bisq.desktop.components.table.BisqTableColumn;
+import bisq.desktop.components.table.IndexColumnUtil;
+import bisq.desktop.components.table.RichTableView;
+import bisq.desktop.main.content.bisq_easy.open_trades.trade_state.states.FormUtils;
+import bisq.desktop.main.content.settings.SettingsViewUtils;
+import bisq.http_api.web_socket.RemoteControlClient;
+import bisq.i18n.Res;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.collections.transformation.SortedList;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.StackPane;
+import javafx.scene.layout.VBox;
+import javafx.scene.text.TextAlignment;
+import javafx.util.StringConverter;
+import lombok.extern.slf4j.Slf4j;
+import org.fxmisc.easybind.EasyBind;
+import org.fxmisc.easybind.Subscription;
+
+@Slf4j
+public class BisqConnectView extends View<VBox, BisqConnectModel, BisqConnectController> {
+    private final Switch enableSwitch;
+    private final AutoCompleteComboBox<BisqConnectHostNetwork> modeComboBox;
+    private final MaterialPasswordField passwordField;
+
+
+    private final MaterialTextField connectionUrlField;
+    private final ImageView qrImageView;
+    private final Label qrPlaceholder;
+    private final Button saveChangesButton;
+
+    private Subscription websocketInitErrorSub;
+
+    BisqConnectView(BisqConnectModel model, BisqConnectController controller) {
+        super(new VBox(35), model, controller);
+        root.setPadding(new Insets(0, 40, 20, 40));
+        root.setAlignment(Pos.TOP_LEFT);
+
+        Label headline = SettingsViewUtils.getHeadline(Res.get("settings.bisqConnect.headline"));
+        Label description = new Label(Res.get("settings.bisqConnect.description"));
+        description.getStyleClass().addAll("normal-text", "wrap-text", "text-fill-grey-dimmed");
+        VBox.setMargin(description, new Insets(5, 0, 10, 0));
+
+        enableSwitch = new Switch(Res.get("settings.bisqConnect.enable"));
+        passwordField = new MaterialPasswordField(
+                Res.get("settings.bisqConnect.password.label"),
+                Res.get("settings.bisqConnect.password.prompt")
+        );
+        passwordField.setHelpText(null);
+        passwordField.setPrefWidth(SettingsViewUtils.TEXT_FIELD_WIDTH);
+        saveChangesButton = new Button(Res.get("settings.bisqConnect.saveChanges"));
+        saveChangesButton.getStyleClass().addAll("grey-transparent-outlined-button", "font-size-12");
+
+        ObservableList<BisqConnectHostNetwork> modes = FXCollections.observableArrayList(BisqConnectHostNetwork.values());
+        modeComboBox = new AutoCompleteComboBox<>(modes, Res.get("settings.bisqConnect.exposure.label"));
```

Comment:
exposure/modeComboBox - Better keep naming of UI element and i18n string synced so its easier to associate.